api/allIncidents/adminIncidentsRouter.js

```diff
@@ -174,8 +174,8 @@ router.post(
   validateAndSanitizeIncidentObject,
   async (req, res, next) => {
     Incidents.createIncident(req.sanitizedIncident)
-      .then((newIncident) => {
-        res.status(201).json(newIncident);
+      .then(() => {
+        res.status(201).json({ message: 'Incident Successfully Created' });
```

Comment:
Will the client not want to receive the new incident object with with it's newly assigned incident id and to see the empty fields?

Reply:
The old model responded with "New Incident Created'. 
Also I would think once admin creates a new incident, all incidents would be grabbed from the server again, including the newly created which would have the next primary key

Frontend might need to make a better form with validation for making a new incident.

Reply:
:)